tests/operator/operator.go

```diff
@@ -2663,16 +2663,14 @@ spec:
 
 				By("Modifying an instancetype")
 				originalInstancetype := instancetypes.Items[0].DeepCopy()
-				instancetype := originalInstancetype.DeepCopy()
-				instancetype.Annotations[keyTest] = valModified
-				instancetype.Labels[keyTest] = valModified
-				instancetype.Spec = v1beta1.VirtualMachineInstancetypeSpec{
-					CPU: v1beta1.CPUInstancetype{
-						Guest: cpu,
-					},
-				}
+				patchDataBytes, err := patch.New(
+					patch.WithAdd(fmt.Sprintf("/metadata/annotations/%s", keyTest), valModified),
+					patch.WithAdd(fmt.Sprintf("/metadata/labels/%s", keyTest), valModified),
+					patch.WithReplace("/spec/cpu", v1beta1.CPUInstancetype{Guest: cpu}),
+				).GeneratePayload()
+				Expect(err).ToNot(HaveOccurred())
 
-				instancetype, err = virtClient.VirtualMachineClusterInstancetype().Update(context.Background(), instancetype, metav1.UpdateOptions{})
+				instancetype, err := virtClient.VirtualMachineClusterInstancetype().Patch(context.Background(), originalInstancetype.Name, types.JSONPatchType, patchDataBytes, metav1.PatchOptions{})
```

Comment:
So while PATCH is surely a better, more precise operation I think using it here may hide conflicts that should not be occurring.
IIUC, the test enables the feature, instancetypes get deployed, and from that point on they should not be changing at all.

Reply:
Not sure I agree here. Here's why: If we use Patch and there's a "conflict" (not as in "returns a conflict error" which will not happen now, but rather as "someone else did another change at the same time and his change won"), then won't the `Except` right after the Patch occasionally fail?
IMO You get the same end result, only you're using a more proper tool.

Reply:
> then won't the `Except` right after the Patch occasionally fail?

It won't fail if that other entity changed different fields on the CR.
In that case, everything will still go through, and we'll never know that the instacetype CR is hotlooping over some field

Reply:
So you want to keep the update in order to *make sure* no other entity modifies it as well?
Are you sure that's the best way to do that?
How can you ensure that if it fails on conflict in the future, that the person trying to fix it won't just put retryOnConflict?